scripts/kconfig/hardened.csv

```diff
@@ -0,0 +1,81 @@
+HW_STACK_PROTECTION,y
+INIT_STACKS,n
+KERNEL_DEBUG,n
+BOOT_BANNER,n
+BOOT_DELAY,0
+EXECUTION_BENCHMARKING,n
+THREAD_MONITOR,n
+THREAD_NAME,n
+STACK_CANARIES,y
+EXECUTE_XOR_WRITE,y
+STACK_POINTER_RANDOM,100
+BOUNDS_CHECK_BYPASS_MITIGATION,y
+PERFORMANCE_METRICS,n
+STATS,n
+DEBUG,n
+TRACING,n
+STACK_USAGE,n
+PRINTK,n
+EARLY_CONSOLE,n
+ASSERT,n
+OBJECT_TRACING,n
+OVERRIDE_FRAME_POINTER_DEFAULT,y
+DEBUG_INFO,n
+OPENOCD_SUPPORT,n
+TRACING_CPU_STATS,n
+TRACING_CTF,n
+USE_SEGGER_RTT,n
+LOG,n
+SHELL,n
+TEST_RANDOM_GENERATOR,n
+ZTEST,n
+TEST,n
+TEST_SHELL,n
+TEST_EXTRA_STACKSIZE,0
+TEST_USERSPACE,n
+CUSTOM_RODATA_LD,n
+CUSTOM_RWDATA_LD,n
+CUSTOM_SECTIONS_LD,n
+BUILD_OUTPUT_STRIPPED,y
```

Comment:
This does not harden the program, based on my understanding of what hardening means ...

Reply:
Nevermind, I can understand that this hardens the program. It makes it more difficult for attackers to develop exploits based on reading the binary.